bwe/RateController.cpp

```diff
@@ -13,6 +15,33 @@ namespace bwe
 {
 const uint32_t ntp32Second = 0x10000u;
 
+uint32_t calculateTargetQueue(double bandwidthKbps, uint32_t minRttNtp, const RateControllerConfig& config)
```

Comment:
If this is local to this cpp file, could you move it to the anonymous namespace instead?